doc/source/train/distributed-pytorch/monitoring-logging.rst

```diff
@@ -16,15 +16,42 @@ be logged and displayed.
 The primary use-case for reporting is for metrics (accuracy, loss, etc.) at
 the end of each training epoch.
 
-.. code-block:: python
+.. tab-set::
+
+    .. tab-item:: PyTorch
+
+        .. code-block:: python
+
+            from ray import train
+
+            def train_func():
+                ...
+                for i in range(num_epochs):
+                    result = model.train(...)
+                    train.report({"result": result})
+
+    .. tab-item:: PyTorch Lightning
```

Comment:
Maybe add one sentence "In PyTorch Lightning, we use a callback to call `train.report()`

Reply:
done